src/adapters/store.js

```diff
@@ -5,35 +5,59 @@ import { getKey } from 'src/libs/pois';
 import { fire } from 'src/libs/customEvents';
 import { isPoiCompliantKey } from 'src/libs/pois';
 
-async function get(k) {
+const prefix = `qmaps_v${version}_`;
+
+function get(k) {
   try {
-    return JSON.parse(localStorage.getItem(k));
+    const prefixedKey = `${prefix}${k}`;
+    return JSON.parse(localStorage.getItem(prefixedKey));
   } catch (e) {
     Error.sendOnce('local_store', 'get', `error parsing item with key ${k}`, e);
     return null;
   }
 }
 
-async function set(k, v) {
+function set(k, v) {
   try {
-    localStorage.setItem(k, JSON.stringify(v));
+    const prefixedKey = `${prefix}${k}`;
+    localStorage.setItem(prefixedKey, JSON.stringify(v));
   } catch (e) {
     Error.sendOnce('local_store', 'set', 'error setting item', e);
   }
 }
 
+function del(k) {
+  const prefixedKey = `${prefix}${k}`;
+  localStorage.removeItem(prefixedKey);
+}
+
+/**
+ * List keys without prefix.
+ * In case some keys are not prefixed, we don't return them
+ */
+function listKeys() {
+  return Object
+    .keys(localStorage || {})
+    .reduce((acc, k) => {
```

Comment:
`reduce` is not easy to read for this use case. Using `filter` and `map` would be easier to understand as it corresponds more closely to the actual operations you do : 1. taking only keys with the prefix, 2. removing the prefix part.

Reply:
actually, it was my first implementation :smile: 
I rewrite it more properly, it's easier to read, thanks